lib/ckb/key.rb

```diff
@@ -54,8 +54,8 @@ def self.random_private_key
 
     def self.pubkey(privkey)
       privkey_bin = [privkey[2..-1]].pack("H*")
-      pubkey_bin = Secp256k1::PrivateKey.new(privkey: privkey_bin).pubkey.serialize
-      Utils.bin_to_hex(pubkey_bin)
+      pubkey_bin = Secp256k1::PrivateKey.from_data(privkey_bin)
+      Utils.bin_to_hex(pubkey_bin.compressed)
```

Comment:
It should create context first and then call the method through key_pair, otherwise an error will occur.
`NoMethodError: undefined method `compressed' for #<Secp256k1::PrivateKey:0x00007fbb06a81768>`

eg:
```
context = Secp256k1::Context.create
privkey_bin = Utils.hex_to_bin(privkey)
key_pair = @context.key_pair_from_private_key(@privkey_bin)
pubkey = Utils.bin_to_hex(@key_pair.public_key.compressed)
```